sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala

```diff
@@ -150,25 +150,27 @@ class JDBCImport(
   private def readJDBCSourceInParallel(): DataFrame = {
 
     if (importConfig.splitBy.nonEmpty) {
+      val defaultString="defaultValue"
       val (lower, upper) = spark.read
         .jdbc(buildJdbcUrl, importConfig.boundsSql, jdbcParams)
-        .selectExpr("cast(min as long) min", "max")
-        .as[(Option[Long], Option[Long])]
+        .selectExpr("cast(min as string) min", "cast(max as string) max")
+        .as[(Option[String], Option[String])]
         .take(1)
-        .map { case (a, b) => (a.getOrElse(0L), b.getOrElse(0L)) }
+        .map { case (a, b) => (a.getOrElse(defaultString), b.getOrElse(defaultString)) }
         .head
 
-      spark.read
-        .jdbc(
-          buildJdbcUrl,
-          importConfig.jdbcQuery,
-          importConfig.splitColumn,
-          lower,
-          upper,
-          importConfig.chunks,
-          jdbcParams
-        )
-        .where(s"${importConfig.splitColumn} >= $lower and ${importConfig.splitColumn} <= $upper")
+      spark.read.format("jdbc")
+        .option("url",buildJdbcUrl)
+        .option("dbtable",importConfig.jdbcQuery)
```

Comment:
double check this. is it a table name or a query.

Reply:
<img width="1000" alt="Screenshot 2022-09-09 at 3 36 48 PM" src="https://user-images.githubusercontent.com/112069583/189327101-6599cbb4-a427-43d6-8e23-06dd62297011.png">
Its the Input table name

Reply:
<img width="1344" alt="Screenshot 2022-09-09 at 4 13 47 PM" src="https://user-images.githubusercontent.com/112069583/189332932-15fc1299-8d12-45e0-9555-14efa1e2c21e.png">

Yes, it can be table name also and can be a query to. As getorElse is used.
But if a query is passed in that parameter this is the error I am getting on my local setup. Hence according to it ,it should always be a table name otherwise the code will produce error

Reply:
<img width="1000" alt="Screenshot 2022-09-09 at 3 36 48 PM" src="https://user-images.githubusercontent.com/112069583/189333556-2a290f3f-32a5-47ff-ae70-45470b813df8.png">

Reply:
In the earlier code too, we can't pass a query(we need table name) when we are passing partition column, upperBound and lowerBound. As I was getting the same error for that code.